### PR TITLE
Remove ms.service value: azure-arc-data

### PR DESCRIPTION
**Pull request by paweenatongbai**
## Description:
This PR is part of a global effort to improve metadata quality. These changes make reports cleaner, easier to use, more consistent, and more reflective of current business.

**Context:**
@C+E Skilling Content Classification is in the process of cleaning up the [ms.prod, ms.technology, ms.service, and ms.subservice allowlists](https://review.docs.microsoft.com/en-us/help/platform/metadata-taxonomies?branch=main). This includes removing bad/invalid/unused values.
Values must be removed from all content in all repos before we can remove them from allowlists.

## Change:
https://dev.azure.com/ceapex/Content%20Learning%20Content%20Architecture/_workitems/edit/568887

Replace: **ms.service: azure-arc-data**
With:
**ms.service: active-directory
ms.subservice: msi**

### CODE REVIEW

**Comment by paweenatongbai** [comment]
@joshfree @weidongxu-microsoft 
I made an update to the metadata per https://dev.azure.com/ceapex/Content%20Learning%20Content%20Architecture/_workitems/edit/568887
Could you please help to approve if you are okay with the change or prefer "ms.service: azure-arc-data" to be removed with no replacement. Thanks!

File: docs-ref-services/latest/resourcemanager-msi-readme.md [hunk 1]
```diff
@@ -6,7 +6,8 @@ ms.author: weidxu
 ms.date: 06/24/2022
 ms.topic: reference
 ms.devlang: java
-ms.service: azure-arc-data
+ms.service: active-directory
```
Comment (joshfree) [inline docs-ref-services/latest/resourcemanager-msi-readme.md:9]: should the keywords on line 3 also be updated?

Comment (paweenatongbai) [reply docs-ref-services/latest/resourcemanager-msi-readme.md:9]: I just saw "\managed-service-identity.md" changed to  ms.service: managedserviceidentity
I should change "\resourcemanager-msi-readme.md" to the same value?

Comment (weidongxu-microsoft) [reply docs-ref-services/latest/resourcemanager-msi-readme.md:9]: @soma-ms

I believe this file is updated by doc stage in SDK release CI.

Does the change in PR stays, or do we need to modify another configure file?

**Review by weidongxu-microsoft — APPROVED** [review]

File: docs-ref-services/latest/resourcemanager-msi-readme.md (hunk 1 above)
Comment (soma-ms) [reply docs-ref-services/latest/resourcemanager-msi-readme.md:9]: @weidongxu-microsoft - I don't have context on this repository. You probably want to tag someone else here.

Comment (sima-zhu) [reply docs-ref-services/latest/resourcemanager-msi-readme.md:9]: This is not the place to change ms.service.
Please submit PR in https://github.com/Azure/azure-sdk/blob/main/_data/releases/latest/java-packages.csv
The `MSDocService` attribute is for updating the value of `ms.service` here.

Comment (paweenatongbai) [reply docs-ref-services/latest/resourcemanager-msi-readme.md:9]: Thanks all. I'll close this PR without merging

**Pull request closed without merging**